playback/core/src/main/kotlin/queue/item/QueueEntry.kt

```diff
@@ -0,0 +1,20 @@
+package org.jellyfin.playback.core.queue.item
+
+// Queue Items
+sealed interface QueueEntry {
+	val skippable: Boolean
+	val metadata: QueueEntryMetadata?
+}
+
+/**
+ * Branded queue items are used for intros, trailers, advertisements and other related videos
+ * It is not possible to seek or skip these items and are normally invisible when showing a queue's contents
+ */
+abstract class BrandedQueueEntry : QueueEntry {
+	override val skippable = false
+}
```

Comment:
No default `metadata` override?

Reply:
This looks like a leftover from an earlier queue concept. We'll probably set the skippable property in the BaseItemDtoUserQueueEntry based on it's BaseItemDto.Type value in the future and remove UserQueueEntry and BrandedQueueEntry entirely.

I'll keep it like this for now and fix change it in an upcoming PR.

Reply:
BrandedQueueEntry and UserQueueEntry now behave the same. I've also added some more comments and made some small adjustments.